.gitignore

```diff
@@ -1,2 +1,4 @@
 # MAC Files
 .DS_Store
+
+node_modules
```

Comment:
Why is node_modules open in the ./ dir?

Reply:
That's a good catch nick, it seems like there's also a package-lock.json in the root dir, not sure why. @gurleen-sanghera , just remove the package-lock and the node_modules folder.

Reply:
I removed the package-lock, but don't see node_modules folder in root dir

Reply:
🤦 ofc it's not gonna be there, it's in the .gitignore lmao, that's my bad. Should be fine to just remove the package-lock lol.